spec/support/vacancy_helpers.rb

```diff
@@ -148,17 +148,15 @@ def verify_vacancy_show_page_details(vacancy)
 
   def verify_vacancy_list_page_details(vacancy)
     expect(page.find('.vacancy')).to have_content(vacancy.job_title)
```

Comment:
We should refactor these into something other than blackbox-full-of-lists-of-test tests.  These don't tell me much about the system and they are at least one level removed from the actual tests.  Also, what happens if the attribute being tested for is null?  Happy to leave it for this PR, but could you put a ticket in to look at it, please?

Reply:
Done